homeassistant/components/zwave_js/api.py

```diff
@@ -183,6 +250,73 @@ async def async_handle_failed_command_func(
     return async_handle_failed_command_func
 
 
+def handle_planned_provisioning_entry(planned_provisioning_entry: dict) -> dict:
+    """Handle provisioning entry as provisioning input."""
+    planned_provisioning_entry = ProvisioningEntry(
+        dsk=planned_provisioning_entry[DSK],
+        security_classes=[
+            SecurityClass(sec_cls)
+            for sec_cls in planned_provisioning_entry[SECURITY_CLASSES]
+        ],
+        additional_properties={
+            k: v
+            for k, v in planned_provisioning_entry.items()
+            if k not in (DSK, SECURITY_CLASSES)
+        },
+    )
+    return planned_provisioning_entry
+
+
+def handle_qr_provisioning_information(qr_provisioning_information: dict) -> dict:
```

Comment:
```suggestion
def handle_qr_provisioning_information(
    qr_provisioning_information: dict
) -> QRProvisioningInformation:
```
Maybe change the parameter name too?